File: src/Assets/ScoreController.js
#pragma strict

import UnityEngine.UI;

private var player : PlayerControll;
private var text : Text;

function Start () {
	player = GameObject.FindGameObjectWithTag("Player").GetComponent.<PlayerControll>();
	text = GetComponent.<Text>();
}

function Update () {
	text.text = player.score.ToString();
}

File: src/Assets/Scripts/ScoreController.cs
using UnityEngine;
using UnityEngine.UI;
using System.Collections;

public class ScoreController : MonoBehaviour {

    Text m_text;

    // Use this for initialization
	void Start () {
        m_text = GetComponent<Text>();
	}
	
	// Update is called once per frame
	void Update () {
        m_text.text = PlayerController.Score.ToString();
	}
}
